ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -175,30 +249,33 @@ export const RuntimePanel = withCurrentWorkspace()(({workspace}) => {
                 value={toolDockerImage}/>
       {/* Runtime customization: change detailed machine configuration options. */}
       <h3 style={styles.sectionHeader}>Cloud compute profile</h3>
-      <FlexRow style={{justifyContent: 'space-between'}}>
-        <MachineSelector updatedMachine={updatedMachine} onChange={setUpdatedMachine} masterMachineType={masterMachineType}/>
-        <DiskSizeSelection updatedDiskSize={updatedDiskSize} onChange={setUpdatedDiskSize} masterDiskSize={masterDiskSize}/>
-      </FlexRow>
+      <div style={styles.formGrid}>
+        <MachineSelector updatedMachine={updatedMachine} onChange={setUpdatedMachine} machineType={masterMachineType}/>
+        <DiskSizeSelection updatedDiskSize={updatedDiskSize} onChange={setUpdatedDiskSize} diskSize={masterDiskSize}/>
+      </div>
       <FlexColumn style={{marginTop: '1rem'}}>
         <label htmlFor='runtime-compute'>Compute type</label>
         <Dropdown id='runtime-compute'
                   style={{width: '10rem'}}
-                  disabled={true}
-                  options={['Dataproc cluster', 'Standard VM']}
-                  value={isDataproc ? 'Dataproc cluster' : 'Standard VM'}/>
+                  options={[ComputeType.dataproc, ComputeType.standard]}
```

Comment:
Per the ticket AC: dataproc should **only** be shown for genomics-enabled workspaces.

How? A little bit tricky actually, now that I look at it..

1. use the `withCdrVersions()` HOC
2. join this against the current `workspace.cdrVersionId`
3. check `cdrVersion.hasMicroarrayData`

If that HOC can't be used with RuntimePanel, this could also happen one level up in the help-sidebar. The help-sidebar could then pass `allowDataproc` (or similar) as a prop here.